src/apps/tools/OswAppFlashLight.cpp

```diff
@@ -0,0 +1,88 @@
+#include "./apps/tools/OswAppFlashLight.h"
```

Comment:
It's a really good idea, 
Well, overall, I think it is difficult to fulfill the expected role because it is a function through color and brightness control rather than light(LED).

How about your opinion? @simonmicro @uvwxy

Reply:
in the dark the flashlight is quite effective

Reply:
I also planned to implement this. Good idea! Like :+1: